Library/Homebrew/extend/os/linux/install.rb

```diff
@@ -51,5 +70,23 @@ def symlink_ld_so
       FileUtils.ln_sf ld_so, brew_ld_so
     end
     private_class_method :symlink_ld_so
+
+    def symlink_gcc_libs
+      gcc_optprefix = HOMEBREW_PREFIX/"opt"/OS::LINUX_PREFERRED_GCC_FORMULA
+
+      GCC_RUNTIME_LIBS.each do |library|
+        gcc_library = gcc_optprefix/"lib/gcc"/GCC_VERSION_SUFFIX/library
+        gcc_library_symlink = HOMEBREW_PREFIX/"lib"/library
+        # Skip if the link target doesn't exist.
+        next unless gcc_library.readable?
+
+        # Also skip if the symlink already exists.
+        next if gcc_library_symlink.readable?
+
+        FileUtils.mkdir_p HOMEBREW_PREFIX
+        FileUtils.ln_sf gcc_library, gcc_library_symlink
```

Comment:
Any reason we're not using `#install_symlink` here?

Reply:
I wasn't sure if that was available in this part of `brew`.  I'll try to use that instead and see if any additional `require` is needed.

Reply:
We `require "extend/pathname"` in `global.rb`:

https://github.com/Homebrew/brew/blob/13b060ab8612e42654fe8fa6004de0d28335fc23/Library/Homebrew/global.rb#L116

So it should be fine.

Reply:
While `install_symlink` does work with no errors, it seems that rather than pointing to the `opt_dir`, it points to the `realpath`.  By contrast, `FileUtils.ln_sf` will link to the `opt_dir` as intended.  I was hoping to use the `opt_dir` because that will be unaffected by revision bumping, so I think there is a now a good reason to prefer `FileUtis.ln_sf` here.